iquip/explorer.py

```diff
@@ -0,0 +1,33 @@
+"""
+App module for showing the experiment list and opening an experiment.
```

Comment:
Although we have used this form in `qiwis`, I think we should write it just like the function or class docstrings, for example:
```python
"""App module for ... an experiment."""
```
or
```python
"""App module for ... blah blah.

More description blah blah.
"""
```

Reply:
Your opinion is more appropriate! I will adopt the former.